composer.json

```diff
@@ -4,19 +4,23 @@
     "keywords": ["doctrine", "aes256", "encrypt", "decrypt"],
     "license": "MIT",
     "description": "Package encrypts and decrypts Doctrine fields through life cycle events.",
-
     "require": {
-        "php": ">=5.3.2",
-        "doctrine/orm": "~2.0",
-        "ext-mcrypt": "*"
+        "php": "^7.1",
+        "doctrine/orm": "^2.5"
     },
-
     "require-dev": {
-        "phpunit/phpunit": "~3.7"
+        "illuminate/encryption": "^5.4",
```

Comment:
Since Laravel 5.4 is the latest version at the moment I don't think that requiring a lower version would make any sense. It doesn't makes sense to support it. This update will be a major new version so requiring the latest Laravel version makes sense.

Adding install notes for `illuminate/encryption` is unnecessary because people who will be using this in the Laravel framework will have the library installed anyway.